cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,25 @@
 /// <reference types='cypress' />
-/// <reference types='../support' />
+
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
```

Comment:
Ensure that the `db:clear` task is correctly implemented to avoid any unintended side effects, such as clearing necessary data for the tests.